curricula/models.py

```diff
@@ -166,7 +166,11 @@ def jackfrost_urls(self):
         return urls
 
     def jackfrost_can_build(self):
-        return settings.ENABLE_PUBLISH and self.status == 2 and not self.login_required
+        can_build = settings.ENABLE_PUBLISH and self.status == 2 and not self.login_required and not self.curriculum.login_required
```

Comment:
So before this change, the `Curriculum`, `Unit`, and `Chapter` classes each had their own `jackfrost_can_build` method, and the method in `Unit` differed from the other two by also checking `self.curriculum`. This change doesn't look right to me because the `Curriculum` class itself doesn't have a `curriculum` field, right? (Or does it somehow?)

Reply:
good catch! I thought both functions were in the same class. Will bring back the old function. It'd be good to add unit tests too.